homeassistant/helpers/template.py

```diff
@@ -2095,15 +2095,22 @@ def today_at(hass: HomeAssistant, time_str: str = "") -> datetime:
     return datetime.combine(today, time_today, today.tzinfo)
 
 
-def relative_time(hass: HomeAssistant, value: Any) -> Any:
+def relative_time(
+    hass: HomeAssistant, value: Any | datetime, is_future: bool = False, depth: int = 1
```

Comment:
`is_future` is not a good name since the caller does not know if the passed time is in the future or in the past, can we come up with something better? Maybe name it `remaining_time`, `time_until`, `elapsed_time` or something like that?

Reply:
allow_future?

Reply:
No, that's not good.

We're not allowing or not allowing the timestamp to be in the future, we're either calculating time elapsed since some time in the past or we're calculating remaining time until some time in the future.